packages/pocl-job/src/staging/__tests__/__snapshots__/create-transactions.spec.js.snap

```diff
@@ -19,6 +19,7 @@ Object {
           "firstName": "Daniel",
           "lastName": "Cleaver",
           "mobilePhone": "07763 722080",
+          "postalFulfilment": undefined,
```

Comment:
should this be undefined?

Reply:
I don't know, it snuck in after the digital licence work was merged in. I'll double check with Nabeel

Reply:
After a chat with Nabeel (all is okay) and digging into the test data, this was due to the way that the mock for getting permit data was set up. I have updated it to return the correct value and now the flag should be set to `true`